src/typeahead/test/typeahead.spec.js

```diff
@@ -389,6 +389,60 @@ describe('typeahead tests', function () {
       expect($scope.result).toEqual('AL');
       expect(inputEl.val()).toEqual('AL');
     });
+    
+    it('should select on an exact match when set', function () {
```

Comment:
All of these should be grouped under a describe block with the select on an exact match descriptor string.